scripts/run-component-tests.js

```diff
@@ -56,7 +56,15 @@ if (local && (components.length > 1 || all)) {
   process.exit(1);
 }
 
-const bazelBinary = `yarn -s ${watch ? 'ibazel' : 'bazel'}`;
+let bazelBinary = `yarn -s `;
+
+if (watch) {
+  // By default ibazel runs the global Bazel binary, but we want it to use the local one.
+  bazelBinary += `ibazel -bazel_path "${require('@bazel/bazelisk/bazelisk.js').getNativeBinary()}"`;
```

Comment:
This isn't ideal, because it doesn't apply to all usages of `ibazel`. E.g. the dev server command still fails, but I couldn't find a nice way of adding the flag without introducing another utility script.

Reply:
This is not the right solution since it's a bug in `ibazel`. This has been reported in the components-discuss channel, and my workaround plan was to provide a custom wrapper for `ibazel` in general to ensure it uses the `bazelisk` one.

We want `yarn ibazel` to also work as expected until the issue gets fixed upstream.

Reply:
I must've missed the Slack discussion. I think we should at least fix it where we can until it's sorted out upstream. Is there an issue I can link?

Reply:
Oh, I thought you've did this upon the Slack channel. This would be the issue: https://github.com/bazelbuild/bazel-watcher/issues/339. I wanted to get to this on Monday with a solution that also works for `yarn ibazel`. It's a problem for folks that don't have Bazel installed globally too.

Reply:
We can also just use this PR. My plan was to have a wrapper file for `ìbazel` that sets the right path to the Bazel binary. So pretty similiar to what you do here I guess.

Reply:
Alright, up to you whether we should keep this approach or do something different. I did this since the testing script isn't usable at the moment.

Reply:
Well, I think we should definitely expand this to also make it work with `yarn ibazel`. That shouldn't be much effort either. It's up to you whether you want to do that change in this PR, or I can also send a follow-up.